### PR TITLE
WindowsPB: Add Incredibuild Dashboard Configuration Task

### PR DESCRIPTION
**Pull request by steelhead31**
Fixes:  https://github.com/adoptium/aqa-tests/issues/5474

As seen in the above issue, the jwebserver tests fail due to a conflict with a process on port 8000. After investigation, this turns out to be a clash with the incredibuild dashboard service.

This change, reconfigures the Incredibuild dashboard service to use a different port. ( The incredibuild service uses port 30000, so 31000 seemed appropriate for the dashboard. Also having carried out investigation, I cannot find any other services using this port on Windows.

##### Checklist

- [X] commit message has one of the [standard prefixes](https://github.com/adoptium/infrastructure/blob/master/CONTRIBUTING.md#commit-messages)
- [X] playbook changes run through [VPC](https://ci.adoptium.net/view/Tooling/job/VagrantPlaybookCheck/) or [QPC](https://ci.adoptium.net/view/Tooling/job/QEMUPlaybookCheck/) (if you have access)

VPC Completed OK : https://ci.adoptium.net/job/VagrantPlaybookCheck/OS=Win2022,label=vagrant/1959/console

### CODE REVIEW

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Incredibuild/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+- name: Replace APACHE_PORT in incredibuild.conf if file exists
+  win_lineinfile:
+    path: 'C:\Program Files (x86)\IncrediBuild\Dashboard\Apache24\conf\incredibuild.conf'
+    regexp: '^define APACHE_PORT \d+$'
+    line: 'define APACHE_PORT 31000'
+    backup: yes
+  when: incredibuild_conf_file.stat.exists
+
+- name: Stop the IBX Dashboard service if it exists
```
Comment (andrew-m-leonard) [inline ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Incredibuild/tasks/main.yml:30]: I think the name should be "Start ..." ?

**Review by andrew-m-leonard — COMMENTED** [review]
Remember to add to Adoptium project, story points, etc...

File: ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Incredibuild/tasks/main.yml (hunk 1 above)
Comment (steelhead31) [reply ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/Incredibuild/tasks/main.yml]: Yes, it should, fixed :)

**Review by andrew-m-leonard — APPROVED** [review]

**Review by sxa — APPROVED** [review]
I'm ok with this, but do we even need the incredibuild service running?

**Pull request merged**